person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(_age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @parent_permission = parent_permission
+  end
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def can_use_services?
+    of_age? || @parent_permission ? true : false
+  end
+
+  private
+
+  def of_age?
+    @age <= 18
+  end
```

Comment:
- [ ] the condition should be if @age is larger than 18 instead of smaller than.